org.adi.pkexec.osc.policy.cmakein

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN" "http://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd">
+<policyconfig>
+  <vendor>Analog devices</vendor>
```

Comment:
Can this be "Analog **D**evices? or "Analog Devices Inc."?

Reply:
Yes, i can make that change.